modules/ROOT/examples/transactions-example.cxx

```diff
@@ -304,4 +311,32 @@ main(int argc, const char* argv[])
         });
         // #end::rollback[]
     }
+
+    {
+        // #tag::max_bucket_instances[]
+        couchbase::cluster c("couchbase://localhost", "transactor", "mypass", cluster_options().max_bucket_instances(10));
+        auto coll = c.bucket("transact")->default_collection();
+        // #end::max_bucket_instances[]
+
+        // #tag::threads[]
+        std::list<std::thread> threads;
+        std::atomic<int> counter {0};
+        for (int i=0; i<10; i++) {
+          threads.emplace_back([&]() {
+            transactions.run([&](couchbase::transactions::attempt_context& ctx) {
+              std::string id = "doc_a";
+              auto doc = ctx.get(coll, id);
+              auto doc_content = doc.value();
+              doc_content["counter"] = ++counter;
+              ctx.replace(coll, doc, doc_content);
+            });
+          });
+        }
+        for (auto& t: threads) {
+          if (t.joinable()) {
+            t.join();
+          }
+        }
+        //#end::threads[]
```

Comment:
This will work, but two sorts of comment chars is redundant - just use the appropriate one for the language, whether //. #. or even ;; or {!-- --} for certain languages  :see_no_evil: